### ISSUE TITLE
Bug/plugin: Regression in email notification plugin

### ISSUE THREAD

**Issue by pdf** [crowdsecurity/crowdsec#1484]
**Describe the bug**
In change a77b3e2690180f30d27ea48409d8d3c2b323caea additional validation was introduced that breaks previously functional configurations utilizing `auth_type: none`, by enforcing non-empty `smtp_username` and `smtp_password`.

**To Reproduce**
1. Configure `notifications/email.yaml` with:
```yaml
smtp_username:        # Replace with your actual username
smtp_password:        # Replace with your actual password
auth_type: none       # Valid choices are "none", "crammd5", "login", "plain"
```
2. Attempt to start with plugin enabled
4. Fail with error: `api server init: unable to run local API: while loading plugin: while configuring email_default: rpc error: code = Unknown desc = SMTP username is not set`

**Expected behavior**
The `smtp_username` and `smtp_password` properties are not required for `auth_type: none`, validation should pass when they are empty and `auth_type` is `none`.

**Technical Information (please complete the following information):**
 - OS: Ubuntu
 - Version: 20.04

**Comment by sabban** [comment]
Hi,
Thank you for reporting this bug: fix will be included in 1.3.4 that is due very soon.

**Comment by buixor** [comment]
and it's out :smiley:
